modules/renter/contractor/uptime_test.go

```diff
@@ -0,0 +1,131 @@
+package contractor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// uptimeHostDB overrides an existing hostDB so that it always returns
+// isOffline == true for a specified address.
+type uptimeHostDB struct {
```

Comment:
should have a different name - 'uptimeHostDB' doesn't give me the impression that it always returns isOffline == true.